pytest_tldr.py

```diff
@@ -60,6 +61,7 @@ def __init__(self, config, file=None):
         self.verbosity = self.config.option.verbose
         self.xdist = getattr(self.config.option, 'numprocesses', None) is not None
         self.hasmarkup = False
+        self.run_quiet_report_verbose = getattr(self.config.option, 'run_quiet_report_verbose', None)
```

Comment:
Is there a reason this defaults to None, rather than False?

Reply:
I think I misunderstood how options work >< I need to figure out how to register an option